react/game/src/app/components/PrivateRoute/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+const PrivateRoute = ({ component: Component, authenticated, ...rest }) => (
+  <Route
+    {...rest}
+    render={props => (authenticated === true ? <Component {...props} /> : <Redirect to="/" />)}
```

Comment:
Remember don't use arrow functions in render method. Also, connect this component to redux to read the `loggedin` value, which I told you before you will need to save it in redux